src/SearchByTitleForm/SearchByTitleForm.js

```diff
@@ -1,34 +1,42 @@
 import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
 import '../Navbar/Navbar.css';
-import { withRouter } from 'react-router';
 
 class SearchByNameForm extends Component {
-    constructor(props) {
-        super(props);
+    constructor() {
+        super();
         this.state = {
             movieTitle: '',
+            movieID: '',
```

Comment:
The selected movie ID is stored here, this is the ID that comes back from the method in App